src/patternfly/components/Select/select.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="pf-c-select{{#if select--IsExpanded}} pf-m-expanded{{/if}}{{#if select--modifier}} {{select--modifier}}{{/if}}"
+  {{#if select--attribute}}
+    {{{select--attribute}}}
+  {{/if}}>
+  {{!-- hidden spans for a11y --}}
+  {{#if select--IsMultiSelect}}
+    <span id="{{id}}-label" hidden>{{#if select-typeahead--Placeholder}} {{select-typeahead--Placeholder}}{{else}}Choose many{{/if}}</span>
+  {{else}}
+    <span id="{{id}}-label" hidden>{{#if select-typeahead--Placeholder}} {{select-typeahead--Placeholder}}{{else}}Choose one{{/if}}</span>
+  {{/if}}
+      
+  {{!-- create the toggle and all its components --}}
+  {{> select-toggle}}
+
+{{!-- create the dropdown menu part --}}
+  {{#if select--IsTypeahead}}
+    {{> select-typeahead}}
```

Comment:
`{{> select-typeahead}}` doesn't need to be closed; but `{{#> select-typeahead}}{{/select-typeahead}}` does. I think the first is more readable since it's less wordy, and nothing needs to be enclosed so it's not necessary. I'd actually prefer to change the one I missed, `{{#> select-checkbox-groups}}{{/select-checkbox-groups}}` :-) What do you think?